packages/backend-common/package.json

```diff
@@ -46,6 +46,10 @@
     "test:kubernetes": "backstage-cli package test -t KubernetesContainerRunner --no-watch"
   },
   "dependencies": {
+    "@aws-sdk/abort-controller": "^3.272.0",
+    "@aws-sdk/client-s3": "^3.276.0",
+    "@aws-sdk/credential-providers": "^3.276.0",
+    "@aws-sdk/types": "^3.272.0",
```

Comment:
I see that other parts of the project consistently use `^3.208.0`. In reality it's all the way up to `3.276.0` in our yarn.lock, but it would be nice if we could have these be consistent across the board as `^3.208.0`, and then some day we upgrade the entire repo at the same time to some new higher version.

This will also have the effect of making the yarn.lock diff smaller.

Is that OK, or does that older version not build against the code you've added?

Reply:
Yeah, it's OK.